### PR TITLE
Regression test/change user permission

### PR DESCRIPTION
**Pull request by srujan-chada**

### CODE REVIEW

File: cypress/integration/internal/change-user-permissions.spec.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      cy.get('#password').type(Cypress.env('DEFAULT_PASSWORD'));
+      cy.get('.govuk-button.govuk-button--start').click();
+      cy.get('#query').type(EMAIL_ADDRESS);
+      cy.get('.search__button').click();
+      cy.get('.govuk-grid-column-full > .govuk-heading-m').should('have.text', 'Users');
+      cy.get('.govuk-list > li').should('contain.text', EMAIL_ADDRESS);
+      cy.get('.govuk-list .govuk-link').click();
+
+      describe('navigates to the user page', () => {
+        cy.url().should('include', '/status');
```
Comment (timtamimi) [inline cypress/integration/internal/change-user-permissions.spec.js:27]: `/status` may also appear in the URL for other pages. Can you make this more specific please?

E.g. you could add
```
cy.url().should('include', '/user/');
```

**Review by timtamimi — CHANGES_REQUESTED** [review]

File: cypress/integration/internal/change-user-permissions.spec.js [hunk 2]
```diff
@@ -24,15 +24,15 @@ describe('change internal user permissions as B&D user', () => {
...
+        cy.url().should('contain', '/user/');
         cy.contains('Internal').should('be.visible');
         cy.get('.govuk-heading-l').eq(0).should('contain', EMAIL_ADDRESS);
         cy.get('.govuk-heading-l').eq(1).should('have.text', 'Set permissions');
       });
       describe('changes the user permissions and navigates to the success page', () => {
         cy.get('#permission-4').check();
         cy.get('form > .govuk-button').click();
-        cy.url().should('include', '/update-permissions/success');
+        cy.url().should('contain', '/user/');
```
Comment (timtamimi) [inline cypress/integration/internal/change-user-permissions.spec.js:35]: Why did you change this one?

File: cypress/integration/internal/change-user-permissions.spec.js [hunk 3]
```diff
@@ -24,15 +24,15 @@ describe('change internal user permissions as B&D user', () => {
       cy.get('.govuk-list .govuk-link').click();
 
       describe('navigates to the user page', () => {
-        cy.url().should('include', '/status');
+        cy.url().should('contain', '/user/');
```
Comment (timtamimi) [inline cypress/integration/internal/change-user-permissions.spec.js:27]: You seem to have replaced the previous assertion. You should keep both I think.

**Review by timtamimi — CHANGES_REQUESTED** [review]

File: cypress/integration/internal/change-user-permissions.spec.js (hunk 2 above)
Comment (srujan-chada) [reply cypress/integration/internal/change-user-permissions.spec.js:35]: you asked to keep user in your previous comment ? So i replaced. :)

**Review by timtamimi — APPROVED** [review]
